testfm/constants.py

```diff
@@ -1,17 +1,25 @@
-import configparser
+from dynaconf import LazySettings
 
-config = configparser.ConfigParser()
-config.read("testfm.properties")
+settings = LazySettings(
+    envvar_prefix="TesTFM",
+    core_loaders=["YAML"],
+    settings_file=["settings.yaml"],
+    preload=["*.yaml"],
+    includes=["settings.local.yaml", ".secrets.yaml", ".secrets_*.yaml"],
+    envless_mode=True,
+    lowercase_read=True,
+)
+
+RHN_USERNAME = settings.subscription.RHN_USERNAME
```

Comment:
Since you ask for lowercase read in settings defination, access the settings in plain lower case letters as :
```suggestion
RHN_USERNAME = settings.subscription.rhn_username
```
same for all other settings.